frontend/src/components/UploadPreviewTable/UploadPreviewTable.scss

```diff
@@ -1,19 +1,17 @@
 @import "src/styles/variables";
 
-.uploadPreviewTable {
+.uploadPreviewTableContainer {
   max-height: 300 * $px;
+  overflow-y: scroll;
+}
 
-  table,
-  th,
-  td {
-    border: $px solid #acacac;
-  }
+.uploadPreviewTable {
+  width: 100%;
+  border: $px solid #9ba5a1;
```

Comment:
what? it's taken directly from Figma design

Reply:
![image](https://user-images.githubusercontent.com/25207759/111996718-5cb2ac00-8b2b-11eb-9f1b-f5ed7e335cb0.png)
Them wrong border color on other elements (it is from second pr)

Reply:
other borders will be fixed in follow-ups of EDNA-31. as you can see there are more even discrepancies with the Figma mock up, but table is going to be done within this PR because I focused on it in this PR.